### PR TITLE
Add DateOnly to Docs #51045

### LINKED ISSUE #51045: Support DateOnly with InputDate
According to the docs, `InputDate` currently supports `DateTime` and `DateTimeOffset`. Should it support `DateOnly` as well?

### PR DESCRIPTION
**Pull request by londonjob**
Add DateOnly to Docs #51045

Fixes #51045

### CODE REVIEW

File: src/Components/Web/src/Forms/InputDate.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ namespace Microsoft.AspNetCore.Components.Forms;
 
 /// <summary>
 /// An input component for editing date values.
-/// Supported types are <see cref="DateTime"/> and <see cref="DateTimeOffset"/>.
+/// Supported types are <see cref="DateTime"/>, <see cref="DateOnly"/> and <see cref="DateTimeOffset"/>.
```
Comment (danmoseley) [inline src/Components/Web/src/Forms/InputDate.cs:13]: TimeOnly?

Comment (Eilon) [reply src/Components/Web/src/Forms/InputDate.cs:13]: Indeed, the code in this file seems to indicate that's also supported, but I'm not an expert in this area.

Comment (danmoseley) [reply src/Components/Web/src/Forms/InputDate.cs:13]: I'm not either but I just looked at the file 😀 let's put both

Comment (londonjob) [reply src/Components/Web/src/Forms/InputDate.cs]: good catch, seems about right. Merci

**Comment by mkArtakMSFT** [comment]
Thanks for your PR, @londonjob.
This is already being addressed as part of another PR we plan to take into 8.0 GA.
@MackinnonBuck can you please link your PR which also includes this fix? Thanks!

**Pull request closed without merging**